app/src/main/java/com/sdp13epfl2021/projmag/database/UserDataDatabase.kt

```diff
@@ -81,4 +80,33 @@ interface UserDataDatabase {
         onSuccess: () -> Unit,
         onFailure: (Exception) -> Unit
     )
-}
\ No newline at end of file
+
+    /**
+     * Apply to the given project by pushing it to
+     * Call `onSuccess` if the operation succeeded
+     * Call `onFailure` with an Exception in case of failure
+     *
+     * @param applied function applies if true, else unapplies
+     * @param onSuccess called on success
+     * @param onFailure called with an exception on failure
+     */
+    fun applyUnapply(
+        applied: Boolean,
```

Comment:
So, if applied is true it should add the projectId,
and if applied is false it will remove it ?